dex-core/src/ErgoDex/Index/ExecutedOrder.hs

```diff
@@ -0,0 +1,120 @@
+module ErgoDex.Index.ExecutedOrder where
+
+import ErgoDex.Amm.Orders
+import ErgoDex.Contracts.Types as Currencies
+import ErgoDex.State
+import ErgoDex.Amm.Pool
+import ErgoDex.Class
+import ErgoDex.Types
+
+import CardanoTx.Models
+
+import           Data.Aeson (FromJSON, ToJSON)
+import qualified PlutusTx.Prelude as P    
+import           Ledger           
+import           GHC.Generics                (Generic)
+
+data ExecutedSwap = ExecutedSwap
+  { config       :: Swap
+  , actualQuote  :: Amount Quote
+  , orderInputId :: String
+  , userOutputId :: String
+  , poolOutputId :: String
+  , poolInputId  :: String
+  } deriving (Show, Eq, Generic, FromJSON, ToJSON)
```

Comment:
Why putting models like this to core lib?

Reply:
moved into specific service